jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/GradleProjectProperties.java

```diff
@@ -102,8 +95,56 @@ static Path getExplodedWarDirectory(Project project) {
     return project.getBuildDir().toPath().resolve(ProjectProperties.EXPLODED_WAR_DIRECTORY_NAME);
   }
 
+  private static EventHandlers makeEventHandlers(
+      Project project, Logger logger, SingleThreadedExecutor singleThreadedExecutor) {
+    LogEventHandlerBuilder logEventHandlerBuilder =
+        (isProgressFooterEnabled(project)
+                ? LogEventHandlerBuilder.rich(singleThreadedExecutor)
+                : LogEventHandlerBuilder.plain(singleThreadedExecutor).progress(logger::lifecycle))
+            .lifecycle(logger::lifecycle);
+    if (logger.isDebugEnabled()) {
+      logEventHandlerBuilder.debug(logger::debug);
+    }
+    if (logger.isInfoEnabled()) {
+      logEventHandlerBuilder.debug(logger::info);
```

Comment:
should this have been `.info(logger::debug)` ?

Reply:
Oh oops, this should have been `.info(logger::info)`